assets/docs/rishit-dagli/plant-disease/1.md

```diff
@@ -0,0 +1,39 @@
+# Module rishit-dagli/plant-disease/1
+Model to classify among 38 plant disease categories.
+
+<!-- module-type: image-classification -->
+<!-- network-architecture: mobilenet -->
+<!-- dataset: plant_village -->
+<!-- fine-tunable: false -->
+<!-- license: Apache-2.0 -->
+<!-- format: saved_model_2 -->
+<!-- asset-path: https://github.com/Rishit-dagli/Greenathon-Plant-AI/releases/download/v0.1.0/saved_model.tar.gz -->
+
+[![Open Colab notebook]](https://colab.research.google.com/github/Rishit-dagli/Greenathon-Plant-AI/blob/main/notebooks/plant-diseases-training.ipynb)
+
+### TF2 SavedModel
+This is a [SavedModel in TensorFlow 2 format](https://www.tensorflow.org/hub/tf2_saved_model). Using it requires TensorFlow 2 (or 1.15) and TensorFlow Hub 0.5.0 or newer.
+
+### Overview
+This model classifies plant leaf images into 38 categories by species and disease. This model was trained on the [plant_village](https://www.tensorflow.org/datasets/catalog/plant_village) (Hughes et al.) dataset [1] on about 87,000 (+ augmented images) healthy and unhealthy leaf images. This model was trained on top of the MobileNet model using weights from MobileNet trained on ImageNet. More info about using this model could be found in the below section.
+
+Here are a couple of images from the dataset on which this model was trained on-
+
+![](https://github.com/Rishit-dagli/Greenathon-Plant-AI/blob/main/media/dataset-examples.png)
+
+### Usage
+This model can be used with the `hub.KerasLayer` as follows.
+
+```py
```

Comment:
Since the model isn't fine-tunable, it might be simpler to demonstrate using this via `hub.load` (eg. `model = hub.load('https://tfhub.dev/rishit-dagli/plant-disease/1')`)

Reply:
Ah that definitely makes more sense, I will make that edit